bot/exts/utilities/challenges.py

```diff
@@ -271,27 +278,25 @@ async def challenge(self, ctx: commands.Context, language: str = "python", *, qu
         if language.lower() not in SUPPORTED_LANGUAGES["stable"] + SUPPORTED_LANGUAGES["beta"]:
             raise commands.BadArgument("This is not a recognized language on codewars.com!")
 
-        get_kata_link = f"https://codewars.com/kata/search/{language}"
+        get_kata_link = f"https://codewars.com/kata/search/{language.lower()}"
```

Comment:
This isn't the only place `language` is used. It would be better IMO to reassign lowercased `language` at the beginning (before line 278) so we can avoid calling `.lower()` multiple times. Further, since `SUPPORTED_LANGUAGES["beta"]` are all lowercase, line 301 wouldn't work for an uppercase language argument :)